lldb/test/API/functionalities/load_unload/TestLoadUnload.py

```diff
@@ -405,8 +405,11 @@ def run_step_over_load(self):
 
     # We can't find a breakpoint location for d_init before launching because
     # executable dependencies are resolved relative to the debuggers PWD. Bug?
+    # The remote lldb server resolves the executable dependencies correctly.
     @expectedFailureAll(
-        oslist=["freebsd", "linux", "netbsd"], triple=no_match("aarch64-.*-android")
+        oslist=["freebsd", "linux", "netbsd"],
+        triple=no_match("aarch64-.*-android"),
```

Comment:
I'm fairly certain you don't need this line. Android testing is always remote, so I expect  that this bit was used as a proxy for "remote=False" (probably unknowingly, because it wasn't known that the remoteness of the config is the important aspect).

Reply:
Updated.